Content.Server/Administration/BwoinkSystem.cs

```diff
@@ -60,15 +97,56 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
             foreach (var channel in targets)
                 RaiseNetworkEvent(msg, channel);
 
+            var sendsWebhook = _webhookUrl != string.Empty;
+            if (sendsWebhook)
+            {
+                var sawmill = IoCManager.Resolve<ILogManager>().GetSawmill("AHELP");
+
+                void LookupFinished(Task<LocatedPlayerData?> finishedTask)
+                {
+                    if (finishedTask.Result == null)
+                    {
+                        sawmill.Log(LogLevel.Error, $"Unable to find player for netuserid {msg.ChannelId} when sending discord webhook.");
+                        return;
+                    }
+
+                    var payload = new WebhookPayload()
+                    {
+                        Username = _serverName,
+                        Content = $"`[{finishedTask.Result.Username}]` {senderSession.Name}: \"{escapedText}\""
```

Comment:
`escapedText` is escaped against our BBCode format. It is completely unapplicable to Discord's markdown format.